src/Blog/View/posts.twig

```diff
@@ -15,8 +22,15 @@
 		<h1>Blog</h1>
 		<ul>
```

Comment:
Good practice is using some code formaring. I just click in editor action: "Format Document", and all rules of code formatting apply automatically. Try do it in you code editor.

Reply:
When I tried to use the shortcut for "Format Document"  I get a pop up that I need an extension to be installed but I don't know which one.

Reply:
Install [Prettier extention](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode)
It  helps with code formating.
And install [Linter extention](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)
Highlights syntax errors in the code.